test/minimal_hello.c

```diff
@@ -1,11 +1,7 @@
-#include <emscripten.h>
-
-EM_JS(void, console_log, (char* str), {
-  console.log(UTF8ToString(str));
-});
+#include <emscripten/console.h>
 
 int main()
 {
-  console_log("minimal hello!");
```

Comment:
I think this will disrupt the minimal_hello to no longer be a minimal hello?

Reply:
Oh nevermind, misinterpreted the line, that is not about the `dbg()`/`err()` code, but goes straight to console.log() as well. LGTM.